demos/improved_qadi_demo.py

```
@@ -0,0 +1,178 @@
+    
+    print(f"Question: {question}")
+    
+    # Direct practical answer
+    print(f"\n📝 DIRECT ANSWER (No QADI):")
+    print("  • Store food properly and plan meals")
+    
+    # QADI-informed answer  
+    print(f"\n🧠 QADI-INFORMED ANSWER:")
+    
+    orchestrator = EnhancedQADIOrchestrator()
+    result = await orchestrator.run_qadi_cycle_with_answers(question)
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)